src/clj_kondo/impl/analyzer/namespace.clj

```diff
@@ -548,6 +548,17 @@
                                ns-name-expr
                                :namespace-name-mismatch
                                (str "Namespace name does not match file name: " ns-name)))))))
+
+        _ (when (and (not (identical? :off (-> ctx :config :linters :underscore-in-namespace :level)))
+                     (symbol? ns-name)
+                     (str/includes? (name ns-name) "_"))
+            (findings/reg-finding!
+             ctx
+             (node->line filename
+                         ns-name-expr
+                         :underscore-in-namespace
+                         (str "Avoid underscore in namespace name: " (namespace-munge ns-name)))))
```

Comment:
Why is `namespace-munge` called here?

Reply:
Oh, my bad, I forgot to remove it when testing the namespace messages. I guess my tests didn't cover this case since they didn't mix `-` and`_` in the namespace names. I'm updating both